chutney/server/src/main/java/com/chutneytesting/index/infra/OnDiskIndexConfig.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.index.infra;
+
+import static com.chutneytesting.tools.file.FileUtils.initFolder;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.apache.lucene.analysis.standard.StandardAnalyzer;
+import org.apache.lucene.index.IndexWriter;
+import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.store.Directory;
+import org.apache.lucene.store.FSDirectory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OnDiskIndexConfig implements  IndexConfig{
+    private final IndexWriter indexWriter;
+    private final Directory indexDirectory;
+
+    public OnDiskIndexConfig(@Value("${chutney.index-folder:~/.chutney/index}") String indexDir) {
+        try {
+            Path path = Paths.get(indexDir);
+            initFolder(path);
+            this.indexDirectory = FSDirectory.open(path);
+            IndexWriterConfig config = new IndexWriterConfig(new StandardAnalyzer());
```

Comment:
Using a standard analyzer is maybe overkill, but the simpliest.

Maybe we could (in another PR), in order to choose an adequate analyzer : 
* Use configuration to list the languages used.
* OR use a tierce library like apache tika to get from document the language used.

Reply:
To elaborate : I think we need stop words of the languages used by the scenario and maybe add some words from the JSON structure itself.
To discuss.